src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -275,6 +275,16 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager', ['bundles' => false]));
```

Comment:
> Are we 100% sure that `@webroot` won't be created ever again with this patch?

With new patch `@webroot` won't be created until significant changes will happen in Yii.

Alternatively we can mock "low-level" protected methods
```php
Stub::make(
    'yii\web\AssetManager',
    [
        'publishFile' => function () {
            return null;
        },
        'publishDirectory' => function () {
            return null;
        },
    ]
);
```
It may be better at some conditions but I think generally it worse because chances that signature of 2 protected methods will be changed are higher than same for 1 public `yii\web\AssetManager::publish`.
Other problem may be overriding of AssetManager by developer in concrete project - if developer overrides AssetManager and code relies on new methods then it is hard to predict what will happen. Then we could use `Maybe` class here.

Also we could specify `basePath` and publish assets:
```php
Stub::make(
    'yii\web\AssetManager',
    [
        'basePath' => '/path/to/webroot/assets',
    ]
);
```
but what should be instead of `/path/to/webroot/assets` is unclear. Should developer specify it in config or it is constant path to `tests/_output/assets` (just example)?